osquery/utils/conversions/windows/windows_time.cpp

```diff
@@ -27,4 +29,38 @@ LONGLONG longIntToUnixtime(LARGE_INTEGER& li) {
   return ull.QuadPart / 10000000ULL - 11644473600ULL;
 }
 
+// Convert little endian Windows FILETIME to unix timestamp
+LONGLONG littleEndianToUnixTime(const std::string& time_data) {
```

Comment:
Are there tests for this?

Reply:
i added a test. let me know if there are any improvements that could be made for it

Reply:
Test looks fine. It shows the thing it's supposed to do.